Jellyfin.Api/Controllers/ImageController.cs

```diff
@@ -1681,7 +1681,7 @@ private static async Task<MemoryStream> GetMemoryStream(Stream inputStream)
             int? width,
             int? height,
             int? quality,
-            bool? cropWhitespace,
+            bool? cropWhitespace, // TODO: Remove
```

Comment:
Can you mark it deprecated in the other functions so it propagates to the OpenAPI spec?

Reply:
I don't see a way to mark it as deprecated. I think the parameter should just be removed if it's just ignored now.

Reply:
@crobibero I know I'm super late to the party, but you can utilize the ObsoleteAttribute class for deprecated/obsolete items:
`[Obsolete("Enter whatever message you'd like.")]`

Reply:
Last time I tested, we weren’t able to put the `Obsolete` attribute on a parameter. You can do it on the entire controller or specific endpoints, which is what we already do.